Site/ASAB Maestro/Descriptors/influxdb.yaml

```diff
@@ -40,6 +40,26 @@ nginx:
       - rewrite ^/influxdb/(.*) /$1 break
       - proxy_pass http://upstream-influxdb
 
+sherpas:
```

Comment:
sherpa runs with every UP. so it must be restartable without breaking antyhing or creating bambilion of new tokens.